scripts/systemd/falco-kmod.service

```diff
@@ -19,7 +19,7 @@ NoNewPrivileges=yes
 ProtectHome=read-only
 ProtectSystem=full
 ProtectKernelTunables=true
-ReadWritePaths=/sys/module/falco
+ReadWriteDirectories=/sys/module/falco
```

Comment:
Thanks!
@Andreagit97 we will need a quick test of the falco-kmod service after this change (from packages built by this pr ci)
This should work with super old versions of systemd as well (it was not working before 231).